File: backend/models/LichSuGiaoHang.js
const { DataTypes } = require("sequelize");
const sequelize = require("../config/db");

const LichSuGiaoHang = sequelize.define(
  "LichSuGiaoHang",
  {
    id: { type: DataTypes.BIGINT, primaryKey: true, autoIncrement: true },
    don_hang_id: { type: DataTypes.BIGINT, allowNull: false },
    tieu_de: { type: DataTypes.STRING(255) },
    mo_ta: { type: DataTypes.STRING(1000) },
    thoi_gian: { type: DataTypes.DATE, defaultValue: DataTypes.NOW },
    lat: { type: DataTypes.DECIMAL(10, 7), allowNull: true },
    lng: { type: DataTypes.DECIMAL(10, 7), allowNull: true },
  },
  {
    tableName: "LichSuGiaoHang",
    timestamps: false,
  }
);

module.exports = LichSuGiaoHang;

File: backend-csharp/Models/LichSuGiaoHang.cs
using System;
using System.Collections.Generic;

namespace WebFashion.Api.Models;

public partial class LichSuGiaoHang
{
    public long Id { get; set; }

    public long DonHangId { get; set; }

    public string? TieuDe { get; set; }

    public string? MoTa { get; set; }

    public DateTimeOffset? ThoiGian { get; set; }

    public decimal? Lat { get; set; }

    public decimal? Lng { get; set; }

    public virtual DonHang DonHang { get; set; } = null!;
}
